### ISSUE TITLE
_CN equivalent implementation

### ISSUE THREAD

**Issue by KameniAlexNea** [phquang/Continual-Normalization#2]
Hi,

I'm trying to use this implementation in my architecture (Class Incremental Learning), and I have a doubt regarding the CN layer proposed.

Since GN and BN are not sharing weights, can we split these into two layers? [CN](https://github.com/phquang/Continual-Normalization/blob/bd3f58064caebff8fa53039a6bc3129dc08fc650/mammoth/models/utils/cn.py#L29)

Proposed :
```
class _CN(nn.Module):
    def __init__(
        self,
        num_features: int,
        momentum: float = 0.01,
        affine: bool = True,
        n_groups=32,
        name="2d",
        eps: float = 1e-3,
    ):
        super().__init__()
        self.gn = nn.GroupNorm(
            num_channels=num_features, num_groups=n_groups, eps=eps, affine=affine
        )
        self.bn = {"1d": BatchNorm1d, "2d": BatchNorm2d, "3d": BatchNorm3d}[name](
            num_features, eps, momentum, affine
        )

    def forward(self, X):
        return self.bn(self.gn(X))

class CN32BN2(_CN):
    def __init__(
        self,
        num_features: int,
        momentum: float = 0.01,
        affine: bool = True,
        eps: float = 1e-3,
    ):
        super().__init__(
            num_features, momentum=momentum, affine=affine, n_groups=32, name="2d", eps=eps
        )
```

**Comment by phquang** [comment]
Our CN first applies a GN without the affine transformation, followed by the standard BN.
Your implementation needs to set ``affine = False`` in ``self.gn``. The rest seems fine to me.
